samples/zoom-to-limits.html

```diff
@@ -0,0 +1,260 @@
+<!doctype html>
```

Comment:
I opened a PR to lint the samples as well https://github.com/chartjs/chartjs-plugin-zoom/pull/194. After's it's merged you can rebase and use the linter, which will make the review easier since there won't be any style issues